build.gradle.kts

```diff
@@ -264,6 +264,7 @@ mavenPublishing {
 dependencies {
     annotationProcessor(libs.nullaway)
     api(libs.protobuf.java)
+    implementation(libs.protobuf.java.util)
```

Comment:
This dependency pulls in several other transitive dependencies (like gson) - if possible could we write our own version of `Timestamps.toString()` for use in the library to not pull in new dependencies?

We should look to write it using `java.time.*` APIs which are immutable.

Reply:
Done, now it uses the `Instant`/`DateTimeFormatter` API instead.